core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -294,8 +295,8 @@
 
 	private EventHandler eventHandler=null;
 
-	private Set<ProcessState> knownProcessStates;
-	private Map<ProcessState,Set<ProcessState>> targetProcessStates = new HashMap<>();
+	private Set<ProcessState> knownProcessStates = new LinkedHashSet<ProcessState>();
+	private Map<ProcessState,Set<ProcessState>> targetProcessStates = new LinkedHashMap<>();
```

Comment:
Turning this into a LinkedHashMap suggests that the insertion order is significant. Is that the case? Are there iterations over this map that should be in the right order?